src/iuse.cpp

```diff
@@ -6411,20 +6487,769 @@ struct npc_photo_def : public JsonDeserializer, public JsonSerializer {
+            } else {
+                description_figures_status.push_back( name_text_pair );
+            }
+        }
+    }
+
+    // scan for everythin NOT near critters
+    object_names_collection obj_coll = enumerate_objects_around_point( aim_point, 2, aim_point, 2,
+                                       camera_pos, min_visible_volume, false,
+                                       ignored_points, vehicles_recorded );
+
+    std::string photo_text = _( "This is a photo of " );
+
+    bool found_item_aim_point;
+    std::string furn_desc = feature_description_at_point( aim_point, found_item_aim_point, 0_ml ) ;
+    item item = get_top_item_at_point( aim_point, 0_ml );
+    std::string trap_name = trap_name_at( aim_point );
+    std::string ter_name = colorized_ter_name_flags_at( aim_point, {}, {} );
+    std::string field_desc = get_field_description_at( aim_point );
+
+    bool  found_vehicle_aim_point = g->m.veh_at( aim_point ).has_value(),
+          found_furniture_aim_point = !furn_desc.empty();
+    // feature_description_at_point do not update flag if no furniture found, so need to check again
+    if( !found_furniture_aim_point ) {
+        found_item_aim_point = !item.is_null();
+    }
+
+    auto ter_aim = g->m.ter( aim_point );
+    auto furn_aim = g->m.furn( aim_point );
+
+    if( !description_figures_status.empty() ) {
+        std::string names = enumerate_as_string( description_figures_status.begin(),
+                            description_figures_status.end(),
+        []( const std::pair<std::string, std::string> &it ) {
+            return "<color_light_blue>" + it.first + "</color>";
+        } );
+
+        photo.name = names;
+        photo_text += names + ".";
+
+        for( const auto &figure_status : description_figures_status ) {
+            photo_text += "\n\n<color_light_blue>" + figure_status.first + "</color> " + figure_status.second;
+        }
+    } else if( found_vehicle_aim_point ) {
+        auto veh_part_pos = g->m.veh_at( aim_point );
+        auto veh_name = "<color_light_blue>" + veh_part_pos->vehicle().disp_name() + "</color>";
+        photo.name = veh_name;
+        photo_text += veh_name + ".";
+        map_deincrement_or_erase( obj_coll.vehicles, veh_name );
+    } else if( found_furniture_aim_point || found_item_aim_point )  {
+        std::string item_desc = colorized_item_description( item ),
+                    item_name = colorized_item_name( item );
+        if( found_furniture_aim_point ) {
+            furn_desc = trap_name + furn_desc + field_desc;
+            photo.name = furn_desc;
+            photo_text += photo.name + ".";
+            map_deincrement_or_erase( obj_coll.furniture, furn_desc );
+        } else if( found_item_aim_point ) {
+            item_name = trap_name + item_name + field_desc;
+            photo.name = item_name;
+            photo_text += item_name + ". " + string_format( _( "It lies on the %1$s." ),
```

Comment:
Mabye a little picky, but let's not use `%$1s` when we don't need to.  `%s` looks cleaner imo and we also save a couple bytes :P

(this isn't the only place this could be changed)
```suggestion
            photo_text += item_name + ". " + string_format( _( "It lies on the %s." ),
```